src/posit/connect/bundles.py

```diff
@@ -3,9 +3,9 @@
 
 import requests
 
-from typing import List
+from typing import BinaryIO, List
```

Comment:
I don't see this used in the diff, is it needed?

```suggestion
from typing import List
```